core/runtime/bindings.rs

```diff
@@ -130,10 +130,11 @@ pub(crate) fn initialize_context<'s>(
   codegen.push_str(include_str!("bindings.js"));
   _ = writeln!(
     codegen,
-    "Deno.__op__ = function(opFns, callConsole, console) {{"
+    "Deno.__op__ = function(opFns, callConsole, console, isOneByte, fromUtf8, toUtf8) {{"
   );
   if init_mode == InitMode::New {
     _ = writeln!(codegen, "Deno.__op__console(callConsole, console);");
+    _ = writeln!(codegen, "Deno.core = {{ ops: {{}}, asyncOps: {{}}, isOneByte, fromUtf8, toUtf8 }};");
```

Comment:
We can probably remove the code from bindings.js that does the ops/asyncOps init.

Reply:
I'm not sure I understand

Reply:
https://github.com/denoland/deno_core/blob/main/core/runtime/bindings.js#L3